Security/src/ExchangeExtendedProtectionManagement/ExchangeExtendedProtectionManagement.ps1

```diff
@@ -323,11 +323,12 @@ begin {
                     $totalCount = $ExchangeServers.Count
                     $progressParams = @{
                         Activity        = "Prerequisites Check"
-                        Status          = [string]::Empty
+                        Status          = "Running Get-OutlookAnywhere...."
```

Comment:
```suggestion
                        Status          = "Running Get-OutlookAnywhere"
```

Reply:
Don't like the `....` to show that it might take a while?

Reply:
Adding dots doesn't really communicate that. It just looks bad. If you want to tell the user it might take a while, actually say that in the text.